File: JUNGOL/Beginner/#2811 소수와 합성수/소수와 합성수.py
def isPrime(n):
    if n%2 == 0: return False

    for i in range(3, int(n**0.5)+1, 2):
        if n%i == 0: return False

    return True

def main():
    for n in list(map(int, input().split())):
        if n == 1: print("number one")
        elif isPrime(n): print("prime number")
        else: print("composite number")

main()

File: JUNGOL/Beginner/#2811 소수와 합성수/소수와 합성수.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool isPrime(int n){
	if(!(n%2)) return false;
	
	for(int i=3; i<=sqrt(n); i+=2)
		if(!(n%i)) return false;

	return true;
}

int main() {
	for(int i=0; i<5; i++){
		int n; cin>>n;

		if(n == 1) cout<<"number one\n";
		else if(isPrime(n)) cout<<"prime number\n";
		else cout<<"composite number\n";
	}
}
